stglib/sg/wvswb2cdf.py

```diff
@@ -0,0 +1,33 @@
+from ..core import utils
+from . import sgutils
+
+
+def wb_to_cdf(metadata):
+    """
+    Load a raw .wb and .hex file and generate a .cdf file
+    """
+    basefile = metadata["basefile"]
+
+    # Get metadata from .hex file
+    hexmeta = sgutils.read_hex(basefile + ".hex")
+
+    # Append to metadata variable
+    metadata.update(hexmeta)
+
+    # Read in data
+    ds = sgutils.read_wb(basefile + ".wb")
+
+    # Convert pressure from psia to dbar
+    ds["P_1"] = ds.P_1 / 14.503773800722 * 10
```

Comment:
Are we guaranteed that the instrument data will always been in psia?

Reply:
Yes, the instrument only reports in psia.